api/prometheus.go

```diff
@@ -113,134 +211,176 @@ func (hs HostSettings) PrometheusMetric() []prometheus.Metric {
 func (m Metrics) PrometheusMetric() []prometheus.Metric {
 	return []prometheus.Metric{
 		{
-			Name:      "host_metrics_accounts_active",
-			Value:     float64(m.Accounts.Active),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_accounts_active",
+			Value: float64(m.Accounts.Active),
+		},
+		{
+			Name:  "hostd_metrics_accounts_balance",
+			Value: m.Accounts.Balance.Siacoins(),
+		},
+		{
+			Name:  "hostd_metrics_revenue_potential_rpc",
+			Value: m.Revenue.Potential.RPC.Siacoins(),
+		},
+		{
+			Name:  "hostd_metrics_revenue_potential_storage",
+			Value: m.Revenue.Potential.Storage.Siacoins(),
+		},
+		{
+			Name:  "hostd_metrics_revenue_potential_ingress",
+			Value: m.Revenue.Potential.Ingress.Siacoins(),
+		},
+		{
+			Name:  "hostd_metrics_revenue_potential_egress",
+			Value: m.Revenue.Potential.Egress.Siacoins(),
+		},
+		{
+			Name:  "hostd_metrics_revenue_potential_registry_read",
+			Value: m.Revenue.Potential.RegistryRead.Siacoins(),
+		},
+		{
+			Name:  "hostd_metrics_revenue_potential_registry_write",
+			Value: m.Revenue.Potential.RegistryWrite.Siacoins(),
+		},
+		{
+			Name:  "hostd_metrics_revenue_earned_rpc",
+			Value: m.Revenue.Earned.RPC.Siacoins(),
+		},
+		{
+			Name:  "hostd_metrics_revenue_earned_storage",
+			Value: m.Revenue.Earned.Storage.Siacoins(),
+		},
+		{
+			Name:  "hostd_metrics_revenue_earned_ingress",
+			Value: m.Revenue.Earned.Ingress.Siacoins(),
 		},
 		{
-			Name:      "hostd_metrics_accounts_balance",
-			Value:     m.Accounts.Balance.Siacoins(),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_revenue_earned_egress",
+			Value: m.Revenue.Earned.Egress.Siacoins(),
 		},
 		{
-			Name:      "hostd_metrics_revenue_potential_rpc",
-			Value:     m.Revenue.Potential.RPC.Siacoins(),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_revenue_earned_registry_read",
+			Value: m.Revenue.Earned.RegistryRead.Siacoins(),
 		},
 		{
-			Name:      "hostd_metrics_revenue_potential_storage",
-			Value:     m.Revenue.Potential.Storage.Siacoins(),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_revenue_earned_registry_write",
+			Value: m.Revenue.Earned.RegistryWrite.Siacoins(),
 		},
 		{
-			Name:      "hostd_metrics_revenue_potential_ingress",
-			Value:     m.Revenue.Potential.Ingress.Siacoins(),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_pricing_contract_price",
+			Value: m.Pricing.ContractPrice.Siacoins(),
 		},
 		{
-			Name:      "hostd_metrics_revenue_potential_egress",
-			Value:     m.Revenue.Potential.Egress.Siacoins(),
-			Timestamp: time.Now(),
+			Name:  "hostd_metrics_pricing_ingress_price",
```

Comment:
Pricing metrics are provided by the other endpoint. Not sure these are necessary

Reply:
for next commit removed these duplicates, kept the latter.
- hostd_settings_ingress_price and hostd_metrics_pricing_ingress_price
- hostd_settings_egress_price and hostd_metrics_pricing_egress_price
- hostd_settings_baserpc_price and hostd_metrics_pricing_baserpc_price
- hostd_settings_sectoraccess_price and hostd_metrics_pricing_sector_access_price
- hostd_settings_contract_price and hostd_metrics_pricing_contract_price
- hostd_settings_storage_price and hostd_metrics_pricing_storage_price